### PR TITLE
Properly merge added non-default properties

### PR DESCRIPTION
**Pull request by lesaltzm**

### CODE REVIEW

File: src/PAModel/MergeTool/Deltas/ChangeProperty.cs [hunk 1]
```diff
@@ -73,6 +73,9 @@ public override void Visit(BlockNode node, ControlPath context)
                             return;
                         }
                     }
+
+                    // Property wasn't present in base
```
Comment (suskumar-MSFT) [inline src/PAModel/MergeTool/Deltas/ChangeProperty.cs:77]: > // Property wasn't present in base [](http://example.com/codeflow?start=19&length=35)

when does this happen ?

**Review by suskumar-MSFT — APPROVED** [review]
:shipit:

**Review by MikeStall — APPROVED** [review]
:shipit:

File: src/PAModel/MergeTool/Deltas/ChangeProperty.cs (hunk 1 above)
Comment (lesaltzm) [reply src/PAModel/MergeTool/Deltas/ChangeProperty.cs:77]: This is if it was a default-valued property in the base, since we remove those from the IR.

**Pull request merged**